### PR TITLE
Adding pdal-parallelizer recipe

### PR DESCRIPTION
**Pull request by ClementAlba**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by ClementAlba** [comment]
@conda-forge/help-python, ready for review

File: recipes/pdal-parallelizer/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9dc874148c437f42c8df5ee6df4b1a8235681ef55616b6c43e846def6701f00b
+
+build:
+  number: 0
```
Comment (dopplershift) [inline recipes/pdal-parallelizer/meta.yaml:13]: This is pure Python and can be marked as a "noarch" build:
```suggestion
  number: 0
  noarch: python
```

File: recipes/pdal-parallelizer/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  script: "python -m pip install . --no-deps --ignore-installed -vv "
+  entry_points:
+    - pdal-parallelizer = pdal_parallelizer.__main__:main
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
```
Comment (dopplershift) [inline recipes/pdal-parallelizer/meta.yaml:23]: For noarch:
```suggestion
    - python >=3.7
  run:
    - python >=3.7
```

**Review by dopplershift — CHANGES_REQUESTED** [review]
A minor tweak to make this noarch (really cuts down on the builds), but otherwise this is in great shape.

**Comment by ClementAlba** [comment]
@dopplershift all changes have been made!

**Comment by ClementAlba** [comment]
@conda-forge/help-python could you please check it again ?

**Comment by ClementAlba** [comment]
@dopplershift Do you know why it indicate "Changes requested" while I have commit your changes and all the checks have passed ?

File: recipes/pdal-parallelizer/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2c7a6e1a128450f1838e1c4cd9a71048621ceda0338a222e25510299484494ab
+
+build:
+  number: 0
+  noarch: python
+  script: "python -m pip install . --no-deps --ignore-installed -vv "
```
Comment (BastianZim) [inline recipes/pdal-parallelizer/meta.yaml:15]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```
This is sufficient.

File: recipes/pdal-parallelizer/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    - distributed
+    - twine
+    - pandas
+    - matplotlib-base
+    - pyproj
+    - python-pdal
+
+test:
+  imports:
+    - pdal_parallelizer
```
Comment (BastianZim) [inline recipes/pdal-parallelizer/meta.yaml:36]:
```suggestion
    - pdal_parallelizer
  commands:
    - pip check
  requires:
    - pip
```
Tests some more stuff

**Review by BastianZim — CHANGES_REQUESTED** [review]
Just a couple of small things, then this is ready to go 🚀

**Pull request merged**